recipes/cython_gsl/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "CythonGSL" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/9c/62/838c21954ca98851f6d8b3778d45e1b1e8e7af98d6cca1dd951dca6b66ad/{{ name }}-{{ version }}.tar.gz
```

Comment:
Please change to: `https://pypi.io/packages/source/C/CythonGSL/CythonGSL-{{ version }}.tar.gz`

(The new PyPI URLs are horrible.)